modules/extensions/src/views/lite/components/debugger/components/Predictions.tsx

```diff
@@ -23,22 +24,26 @@ const Predictions = (props: Props) => {
       <div className={style.sectionTitle}>Top Predictions</div>
       {Object.keys(predictions).map((key, index) => {
         const { confidence, intents } = predictions[key]
-        return (
-          <div className={style.subSection} key={index}>
-            <p>
-              {key} {formatConfidence(confidence)}%
-            </p>
-            <ul>
-              {intents.slice(0, 4).map(i => {
-                return (
-                  <li key={i.label}>
-                    <Intent name={i.label} confidence={i.confidence} />
-                  </li>
-                )
-              })}
-            </ul>
-          </div>
-        )
+        if ((key === 'global' && activePrompt) || key !== 'global') {
```

Comment:
please extract 'global' in a constant. I have a feeling like we might change this key and will be much easier to change if it's extracted in a constant

Reply:
Can you rename key variable for topicName it'll be much cleaner